abci/multiplexer.go

```diff
@@ -199,21 +448,78 @@ func (m *Multiplexer) getApp() (servertypes.ABCI, error) {
 	return nil, fmt.Errorf("unknown ABCI client version %d", currentVersion.ABCIVersion)
 }
 
+// startEmbeddedApp starts an embedded version of the app.
+func (m *Multiplexer) startEmbeddedApp(version Version) error {
```

Comment:
created util function since it was used in a few places. Code unchanged other than that